packages/tools/pluggable-widgets-tools/bin/mx-scripts.js

```diff
@@ -77,6 +77,8 @@ function getRealCommand(cmd, toolsRoot) {
         case "test:e2e:ts":
         case "test:e2e:js":
             return `wdio ${join(toolsRoot, "test-config/wdio.conf.js")}`;
+        case "test:e2e:dev":
```

Comment:
I think it is better to call it `test:e2e:web:dev`

Reply:
Ok!